src/editor/CSSInlineEditor.js

```diff
@@ -362,8 +382,9 @@ define(function (require, exports, module) {
 
         return result.promise();
     }
-
+    
     EditorManager.registerInlineEditProvider(htmlToCSSProvider);
+    EditorManager.registerEditProvider(checkProvider);
```

Comment:
This seems wrong to me.  Instead of all these gymnastics to expose this, I think you should provide a public method on this module, `checkInlineProvider()` or something, and have code elsewhere include this module and call it at the right time.